codeHF/dpl-config_run3.json

```diff
@@ -513,6 +546,38 @@
             ]
         }
     },
+    "hf-task-ds": {
+        "cutYCandMax": "0.8",
+        "selectionFlagDs": "0",
+        "pTBins": {
+            "values": [
+                "2.",
+                "3.",
+                "4.",
+                "5.",
+                "6.",
+                "8.",
+                "12.",
+                "16."
+            ]
+        }
+    },
+    "hf-task-ds-mc": {
```

Comment:
This part should disappear once the MC part becomes a process function.

Reply:
Hi @vkucera! Ok, I took the D+ as an example and it still has this MC part. Do you want me to remove it anyway?

Reply:
Yes, the change from workflow options to process functions should be propagated everywhere.